diff --git a/libpurple/mediamanager.c b/libpurple/mediamanager.c
--- a/libpurple/mediamanager.c
+++ b/libpurple/mediamanager.c
@@ -2431,7 +2431,7 @@
 	purple_media_manager_register_element(manager,
 		g_object_new(PURPLE_TYPE_MEDIA_ELEMENT_INFO,
 			"id", "autoaudiosrc",
-			"name", N_("Default"),
+			"name", (const char *)_("Default"),
 			"type", PURPLE_MEDIA_ELEMENT_AUDIO
 				| PURPLE_MEDIA_ELEMENT_SRC
 				| PURPLE_MEDIA_ELEMENT_ONE_SRC
@@ -2442,7 +2442,7 @@
 	purple_media_manager_register_element(manager,
 		g_object_new(PURPLE_TYPE_MEDIA_ELEMENT_INFO,
 			"id", "autoaudiosink",
-			"name", N_("Default"),
+			"name", (const char *)_("Default"),
 			"type", PURPLE_MEDIA_ELEMENT_AUDIO
 					| PURPLE_MEDIA_ELEMENT_SINK
 					| PURPLE_MEDIA_ELEMENT_ONE_SINK,
@@ -2452,7 +2452,7 @@
 	purple_media_manager_register_element(manager,
 		g_object_new(PURPLE_TYPE_MEDIA_ELEMENT_INFO,
 			"id", "autovideosrc",
-			"name", N_("Default"),
+			"name", (const char *)_("Default"),
 			"type", PURPLE_MEDIA_ELEMENT_VIDEO
 				| PURPLE_MEDIA_ELEMENT_SRC
 				| PURPLE_MEDIA_ELEMENT_ONE_SRC
@@ -2463,7 +2463,7 @@
 	purple_media_manager_register_element(manager,
 		g_object_new(PURPLE_TYPE_MEDIA_ELEMENT_INFO,
 			"id", "autovideosink",
-			"name", N_("Default"),
+			"name", (const char *)_("Default"),
 			"type", PURPLE_MEDIA_ELEMENT_VIDEO
 				| PURPLE_MEDIA_ELEMENT_SINK
 				| PURPLE_MEDIA_ELEMENT_ONE_SINK,
@@ -2479,7 +2479,7 @@
 			 * possible audio input device. The device can help the
 			 * user to check if her speakers or headphones have been
 			 * set up correctly for voice calling. */
-			"name", N_("Test Sound"),
+			"name", (const char *)_("Test Sound"),
 			"type", PURPLE_MEDIA_ELEMENT_AUDIO
 				| PURPLE_MEDIA_ELEMENT_SRC
 				| PURPLE_MEDIA_ELEMENT_ONE_SRC,
@@ -2489,7 +2489,7 @@
 	purple_media_manager_register_element(manager,
 		g_object_new(PURPLE_TYPE_MEDIA_ELEMENT_INFO,
 			"id", "disabledvideosrc",
-			"name", N_("Disabled"),
+			"name", (const char *)_("Disabled"),
 			"type", PURPLE_MEDIA_ELEMENT_VIDEO
 				| PURPLE_MEDIA_ELEMENT_SRC
 				| PURPLE_MEDIA_ELEMENT_ONE_SINK,
@@ -2504,7 +2504,7 @@
 			 * a test "monoscope" image that can help the user check
 			 * the video output has been set up correctly without
 			 * needing a webcam connected to the computer. */
-			"name", N_("Test Pattern"),
+			"name", (const char *)_("Test Pattern"),
 			"type", PURPLE_MEDIA_ELEMENT_VIDEO
 				| PURPLE_MEDIA_ELEMENT_SRC
 				| PURPLE_MEDIA_ELEMENT_ONE_SRC,
